data_steward/validation/main.py

```diff
@@ -340,14 +349,21 @@ def get_heel_result(hpo_id, app_id=None, dataset_id=None):
     return result
 
 
-def get_heel_error_header(list_of_dicts):
-    fake_header_list = ['Record Count', 'Heel Error', 'Analysis ID', 'Rule ID']
-    if not list_of_dicts:
-        return fake_header_list
+def _convert_query_result_to_row_list(list_of_dicts):
+    row_list = list()
+    if list_of_dicts is []:
+        return row_list.append(tuple())
```

Comment:
This returns the result of the _action of appending itself_, not `row_list`, so the return type will be `None`. 
`row_list` needs to be returned.